hazelcast-sql/src/main/codegen/config.fmpp

```diff
@@ -429,6 +431,7 @@ data: {
       "VIEWS"
       "TYPES"
       "RESOURCES"
+      "ANALYZE"
```

Comment:
This list was alphabetically sorted before the introduction of `TYPES` and `RESOURCES`. Let's make it sorted again.